src/js/containers/search/topFilterBar/TopFilterBarContainer.jsx

```diff
@@ -140,6 +146,30 @@ export class TopFilterBarContainer extends React.Component {
         return null;
     }
 
+
+    /**
+     * Logic for parsing the current Redux keyword filter into a JS object that can be parsed by the
+     * top filter bar
+     */
+    prepareKeywords(props) {
+        let selected = false;
+        const filter = {};
+
+        if (props.keyword) {
+        // keyword exists
+            selected = true;
+            filter.code = 'keyword';
+            filter.name = 'Keyword';
+
+            filter.values = props.keyword;
```

Comment:
We need to figure out how to handle different filter type lengths in `TopFilterBar`. I started this fix in Award Amount, but it applies here too, to String types; the "length" of this filter is technically the length of the keyword string, so typing in "Education" will show "9 Current Filters".